topologies/topology_test/otg_tests/topology_test.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2022 Google LLC
```

Comment:
Sorry if I was not clear before, but the convention for test paths should be:

* ATE: topologies/ate_tests/topology_test/topology_test.go
* OTG: topologies/otg_tests/topology_test/topology_test.go

Reply:
Done.